modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -913,7 +913,7 @@ module.exports = (self) => {
       self.register(metaType, type, field.schema);
     },
     validate: function (field, options, warn, fail) {
-      for (const subField of field.schema || field.fields.add) {
+      for (const subField of field.schema) {
```

Comment:
Removed because `field.fields.add` could never be iterated over since it's an object.